test/passes/flatten-control-flow.bin.txt

```diff
@@ -119,51 +119,7 @@
   (local $var$8 f64)
   (block $label$0
    (nop)
-   (block
```

Comment:
Looks like the goal was to see that unreachable code doesn't break us, there was nothing in need of flattening after the unreachable. So the test is still fulfilling that role. Code truly in need of flattening is tested in $9 after this one, and in the wast (text) test for this.